webpack.config.base.js

```diff
@@ -18,20 +23,12 @@ module.exports = options => ({
       path.resolve(__dirname, baseDir),
       'node_modules'
     ],
-    alias: {
-      'rsg-components/ComponentsList/ComponentsListRenderer': path.join(__dirname, 'styleguide_assets/CustomRenderers/ComponentsListRenderer'),
-      'rsg-components/StyleGuide/StyleGuideRenderer': path.join(__dirname, 'styleguide_assets/CustomRenderers/StyleGuideRenderer'),
-      'rsg-components/ReactComponent/ReactComponentRenderer': path.join(__dirname, 'styleguide_assets/CustomRenderers/ReactComponentRenderer'),
-    }
   },
   module: {
     rules: get(options, 'module.rules', []).concat([
       {
         test: /\.js$/,
-        exclude: [
-          path.resolve(__dirname, 'node_modules'),
-          /__tests__/
-        ],
+        include: includePaths,
```

Comment:
I might change this to `options.includePaths` and just keep the `includePaths` constant in the `dev` config since I'm not sure we need to include `styleguide_assets` for prod.